packages/eslint-plugin/tests/util/misc.test.ts

```diff
@@ -1,5 +1,26 @@
 import * as misc from '../../src/util/misc';
 
+describe('isDefinitionFile', () => {
+  it('returns true for standard definition files', () => {
```

Comment:
[Testing] These tests each have multiple cases in them. Which is generally not a great testing thing: if an earlier assertion fails, then subsequent ones won't get run. That makes for not-very-informative test failures.

Could you please change these to be one assertion per test?

FWIW I've personally enjoyed using `it.each` to make tables of tests when there are a lot of very similar cases.

Reply:
Hello @JoshuaKGoldberg 
Can you confirm that I got your approach in the right way?

```ts
  it.each([
    ['index.d.ts'],
    ['module.d.cts'],
    ['package.d.mts'],
  ])('returns true for standard definition file: %s', filename => {
    expect(misc.isDefinitionFile(filename)).toBe(true);
  });

  it.each([
    ['styles.d.css.ts'],
    ['component.d.vue.ts'],
    ['env.d.node.ts'],
  ])('returns true for arbitrary extension definition file: %s', filename => {
    expect(misc.isDefinitionFile(filename)).toBe(true);
  });

  it.each([
    ['index.ts'],
    ['app.tsx'],
    ['styles.css.ts'],
    ['vite.config.ts'],
  ])('returns false for non definition file: %s', filename => {
    expect(misc.isDefinitionFile(filename)).toBe(false);
  });
```

Reply:
Yup that works!